less/index.less

```diff
@@ -52,8 +52,188 @@ table {
 * {
     box-sizing: border-box;
 }
-
+html{
+	font-size: 62.5%;
+	
+}
 html, body {
-    height: 100%;
+	height: 100%;
+}
+@mainFontSize: 1.6rem;
+@textColor: #424242;
+@hCTextColor: white;
+@fontStack: Helvetica, Ariel, sans-serif;
+
+.greyMainFont(){
+	font-size: @mainFontSize;
+	color: @hCTextColor;
+	font-family: @fontStack;
+}
+.mainFont(){
+	font-size: @mainFontSize;
+	color: @textColor;
+	font-family: @fontStack;
+}
+.subjBoxes() {
+	font-size: 2.8rem;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	padding: 10px;
+	margin-top: 20px;
+	
+}
+
+
+.container {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	padding-bottom: 20px;
+}
+.container .navBar{
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin: 0 auto;
+	padding: 20px;
+	background-color:#fafafa; 
+	width: 100%;
+
+	a {
+		text-decoration: none;
+		font-size:1.6rem;
+		color: @textColor;
+		font-family: @fontStack;
+		padding-left:5%;
+		padding-right:5%;
+	}
+}
+.resumeContainer{
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	
+}
+.resumeContent {
+	width: 70%;
+	padding-top: 8%;
+	padding-left: 10%;
+	margin-right: 8%;
+	display:flex;
+	flex-direction:column;
+	padding-bottom: 20px;
```

Comment:
try adding a little color to the background of your resume content, just to make it pop (look at your already-used colors, like the blue or the green)